src/Persistence.LinnApps/Repositories/PartSupplierRepository.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Linn.Purchasing.Persistence.LinnApps.Repositories
+{
+    using Linn.Common.Persistence.EntityFramework;
+    using Linn.Purchasing.Domain.LinnApps.PartSuppliers;
+    using Linn.Purchasing.Persistence.LinnApps.Keys;
+
+    using Microsoft.EntityFrameworkCore;
+
```

Comment:
I don't think you need this repository declared as it stands although i appreciate it's probably there because you will need to add overrides in the future